.formatter.exs

```diff
@@ -1,4 +1,5 @@
 # Used by "mix format"
 [
+  locals_without_parens: [assert_close: 2],
```

Comment:
Btw, I believe we have `assert_in_delta` in Elixir, but it obviously doesn't work for complex. So just FYI. :)

Reply:
yeah, I considered using that, but it ended up not serving my use case at the time